klaytnetl/mappers/receipt_mapper.py

```diff
@@ -96,6 +98,12 @@ def json_dict_to_receipt(self, json_dict, **kwargs) -> KlaytnRawReceipt:
         )
         receipt.max_fee_per_gas = hex_to_dec(json_dict.get("maxFeePerGas"))
 
+        if self.enrich:
+            receipt = KlaytnReceipt.enrich(
+                receipt,
+                block_timestamp=kwargs.get("block_timestamp")
```

Comment:
How about the field `block_unix_timestamp`?

Reply:
once we enrich block_timestamp, this will be calculated in receipt_to_dict